frontend/src/components/FileUpload.js

```diff
@@ -1,14 +1,17 @@
 import {fileUpload} from "../services/fileUploadService";
 import {useState} from "react";
+import {useHistory} from "react-router-dom";
 
-export default function FileUpload() {
+export default function FileUpload({onDraftReceived}) {
 
     const [file, setFile] = useState(undefined);
+    const history =  useHistory();
 
     const onFormSubmit = (formEvent) => {
         formEvent.preventDefault();
         fileUpload(file).then((response) => {
-            console.log(response.data)
+            onDraftReceived(response);
+            history.push("/draft");
```

Comment:
In case the fileUpload is not working...there is no form reset and/or user notification?

Reply:
The file upload on the frontend is only provisory at the moment to test backend functionality, this will be adressed in a later task